app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/database/dummy/DummyDatabase.java

```diff
@@ -61,4 +63,28 @@ public void reset() {
         conversationsDB.removeAll(conversationsDB);
     }
 
+    @Override
+    public Task<DocumentSnapshot> fetch(String collectionPath, String documentPath) {
+        return null;
+    }
+
+    @Override
+    public Task<Void> set(String collectionPath, String documentPath, Object data) {
+        return null;
+    }
+
+    @Override
+    public Task<Void> delete(String collectionPath, String documentPath) {
+        return null;
+    }
+
+    @Override
+    public Task<String> add(String collectionPath, Object data) {
+        return null;
+    }
+
+    @Override
+    public Task<CollectionSnapshot> fetchAll(String collectionPath) {
+        return null;
+    }
```

Comment:
these methods shouldn't be retuning null, it should be at least returning dummy tasks

Reply:
The dummy db will be implemented correctly later, after discussing with ellior and the rest of the team.